src/ImageProcessor/Colors/Colorspaces/CieXyz.cs

```diff
@@ -0,0 +1,176 @@
+    /// </summary>
+    public struct CieXyz : IEquatable<CieXyz>
+    {
+        /// <summary>
+        /// Represents a <see cref="CieXyz"/> that has Y, Cb, and Cr values set to zero.
+        /// </summary>
+        public static readonly CieXyz Empty = default(CieXyz);
+
+        /// <summary>
+        /// The backing vector for SIMD support.
+        /// </summary>
+        private Vector3 backingVector;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CieXyz"/> struct.
+        /// </summary>
+        /// <param name="y">The y luminance component.</param>
+        /// <param name="x">X is a mix (a linear combination) of cone response curves chosen to be nonnegative</param>
+        /// <param name="z">Z is quasi-equal to blue stimulation, or the S cone of the human eye.</param>
+        public CieXyz(float x, float y, float z)
+            : this()
+        {
+            // ToDo: check clamp values
+            this.backingVector.X = x.Clamp(0, 2);
+            this.backingVector.Y = y.Clamp(0, 2);
+            this.backingVector.Z = z.Clamp(0, 2);
+        }
+
+        /// <summary>
+        /// Gets the Y luminance component.
+        /// <remarks>A value ranging between 380 and 780.</remarks>
+        /// </summary>
+        public float X => this.backingVector.X;
+
+        /// <summary>
+        /// Gets the Cb chroma component.
+        /// <remarks>A value ranging between 380 and 780.</remarks>
+        /// </summary>
+        public float Y => this.backingVector.Y;
+
+        /// <summary>
+        /// Gets the Cr chroma component.
+        /// <remarks>A value ranging between 380 and 780.</remarks>
+        /// </summary>
+        public float Z => this.backingVector.Z;
+
+        /// <summary>
+        /// Gets a value indicating whether this <see cref="CieXyz"/> is empty.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool IsEmpty => this.backingVector.Equals(default(Vector3));
+
+        /// <summary>
+        /// Allows the implicit conversion of an instance of <see cref="Color"/> to a
+        /// <see cref="CieXyz"/>.
+        /// </summary>
+        /// <param name="color">
+        /// The instance of <see cref="Color"/> to convert.
+        /// </param>
+        /// <returns>
+        /// An instance of <see cref="CieXyz"/>.
+        /// </returns>
+        public static implicit operator CieXyz(Color color)
+        {
+            var r = color.R;
+            var g = color.G;
+            var b = color.B;
+
+            // assume sRGB
+            r = r > 0.04045 ? (float)Math.Pow(((r + 0.055) / 1.055), 2.4) : (float)(r / 12.92);
+            g = g > 0.04045 ? (float)Math.Pow(((g + 0.055) / 1.055), 2.4) : (float)(g / 12.92);
+            b = b > 0.04045 ? (float)Math.Pow(((b + 0.055) / 1.055), 2.4) : (float)(b / 12.92);
+
+            var x = (r * 0.41239079926595F) + (g * 0.35758433938387F) + (b * 0.18048078840183F);
```

Comment:
Could we use explicit type declaration here. I find var can sometimes lead to subtle numerical errors that can be hard to debug.